### ISSUE TITLE
Valve no opening with Flow

### ISSUE THREAD

**Issue by greenwell72** [AdyRock/com.linktap#21]
Hi Andy,

I have recently noticed that if I schedule a valve to open via a flow, the app in homey indicates that it has been opened but in fact it hasn't. No water flows and I dont get the notification from the Linktap app. The same happens if I try to manually trigger it in Homey by tapping on the valve icon. 

Any ideas?

**Comment by AdyRock** [comment]
I just checked tapping on the icon and It's working for me. The notification from Linktap can take up to 15 seconds as it waits for the message to be sent to the valve and acknowledged.
I presume it works if you turn it on from the LinkTap app?

**Comment by greenwell72** [comment]
Yeah scheduling and manual triggering works fine in the Linktap app.

I've just installed the latest test version to get the total water readings but can't say right now if it fixes this issue(am traveling now).

Will do as soon as I get home.

**Comment by greenwell72** [comment]
Hi Andy, just let you know that it seems to be working now. Not sure if the issue was just me 😃
